packages/core/src/modules/proofs/models/ProofServiceOptions.ts

```diff
@@ -19,7 +19,7 @@ interface BaseOptions {
 }
 
 export interface CreateProposalOptions<PFs extends ProofFormat[]> extends BaseOptions {
-  connectionRecord: ConnectionRecord
+  connectionRecord?: ConnectionRecord
```

Comment:
Currently all proposals require a connection, so we shouldn't make it optional here. 

```suggestion
  connectionRecord: ConnectionRecord
```